light_controller/ir_toy/ir_toy.py

```diff
@@ -0,0 +1,351 @@
+# -*- coding: utf-8 -*-
+import logging
+import time
+import struct
+
+import serial
+
+from .ir_toy_settings import IR_ToySettings
+
+__author__ = 'akm'
+
+
+class CommandEnum:
+    reset = 0x00
+    SUMP_Run = 0x01
+    SUMP_Id = 0x02
+    transmit = 0x03
+    getFrequencyReport = 0x04
+    setSampleTimer = 0x05
+    setFrequencyModulationTimer = 0x06
+    setLED_MuteOn = 0x10
+    setLED_MuteOff = 0x11
+    setLED_On = 0x12
+    setLED_Off = 0x13
+    getSettingsReport = 0x23
+    enableTX_ByteReport = 0x24
+    enableTX_NotifyOnComplete = 0x25
+    enableTX_Handshake = 0x26
+    IO_Write = 0x30
+    IO_Direction = 0x31
+    IO_Read = 0x32
+    IO_Direction = 0x31
+    setupUART = 0x40
+    closeUART = 0x41
+    writeUART = 0x42
+
+
+class IR_Toy:
+    TERMINATOR = [255, 255]
+
+    """
+    0x00 Reset (returns to remote decoder mode)
+    0x01 RESERVED for SUMP RUN
+    0x02 RESERVED for SUMP ID
+    0x03 Transmit (FW v07+)
+    0x04 Frequency report (reserved for future hardware)
+    0x05 Setup sample timer (FW v07+)
+    0x06 Setup frequency modulation timer (FW v07+)
+    0x10 LED mute on (FW v07+)
+    0x11 LED mute off (FW v07+)
+    0x12 LED on (FW v07+)
+    0x13 LED off (FW v07+)
+    0x23 Settings descriptor report (FW v20+)
+    0x24 Enable transmit byte count report (FW v20+)
+    0x25 Enable transmit notify on complete (FW v20+)
+    0x26 Enable transmit handshake (FW v20+)
+    0x30 IO write
+    0x31 IO direction
+    0x32 IO read
+    0x40 UART setup
+    0x41 UART close
+    0x42 UART write
```

Comment:
Maybe move these into comments on lines of the enum above? Looks like we have the same info twice.